app/services/notices/setup/preview-return-forms.service.js

```diff
@@ -13,12 +13,13 @@ const PrepareReturnFormsService = require('./prepare-return-forms.service.js')
  *
  * This service returns the file to be display in the browser. This will likely be the built-in pdf viewer.
  *
- * @param {string} sessionId
+ * @param {string} sessionId - The UUID of the current session
+ * @param {string} returnId - The UUID of the return id
  *
  * @returns {Promise<ArrayBuffer>} - Resolves with the generated form file as an ArrayBuffer.
  */
-async function go(sessionId) {
-  return PrepareReturnFormsService.go(sessionId)
+async function go(sessionId, returnId) {
+  return PrepareReturnFormsService.go(sessionId, returnId)
```

Comment:
```suggestion
async function go(sessionId, returnLogId) {
  return PrepareReturnFormsService.go(sessionId, returnLogId)
```

Reply:
I am an idiot. We do have a UUID in the table now, and it is called `return_id`. I forgot we made this change 🤦